etc/skate-style/palette.mss

```diff
@@ -129,6 +132,9 @@ Map { font-directory: url(./fonts); }
    at once or override each individually. */
 @place_halo:        fadeout(#fff,34%);
 
+@halo_white:        @place_halo;
+@greenspace_halo:   @halo_white;
+
```

Comment:
This also doesn't feel like the final resting place of these variables, so let me know if you have ideas of where it should be organized into.